### ISSUE TITLE
empty ebuttm:documentRevisionNumber passes syntactic validation

### ISSUE THREAD

**Issue by nigelmegitt** [ebu/ebu-tt-live-toolkit#220]
A document with an empty `ebuttm:documentRevisionNumber` is passed by the validation code but not by Oxygen with an XSD validation. The type is `xs:nonNegativeInteger`.

**Comment by kozmaz87** [comment]
Yes this issue is caused by the default value PyXB implementation details of the type permit. It is a PyXB bug and needs working around

**Comment by kozmaz87** [comment]
Related pyxb issue
https://github.com/pabigot/pyxb/issues/71
